ScottWu.md

```diff
@@ -1,18 +1,18 @@
-## 11/14 ~ 11/18
+## 11/21 ~ 11/25
 
 - [Firefox]
 	- [UI feedback] [Bug 285836](https://bugzilla.mozilla.org/show_bug.cgi?id=285836) - Make "Remove All Cookies Now" undoable
 		- Got UX feedback, will make a few changes to the patch
 
 - [Date-time Picker]
 	- [Bug 1283381](https://bugzilla.mozilla.org/show_bug.cgi?id=1283381) - [meta] Implement UI for date time input types
-		- [WIP] [Bug 1309471](https://bugzilla.mozilla.org/show_bug.cgi?id=1309471) - [DateTimePicker] Add browser chrome test for time picker
-			- Placing it on lower priority, may include the spinner functionality tests in date picker tests instead
-		- [WIP] [Bug 1283385](https://bugzilla.mozilla.org/show_bug.cgi?id=1283385) -  Implement UI for \<input type="date"\>
-			- Improved calendar rendering to only update the changes
-			- Stop updating calendar if month year picker is covering the calendar
-			- Stop updating month year spinners if the month year picker is not visible
-			- DateKeeper reuses days, months, years arrays if possible
+		- [r?] [Bug 1283385](https://bugzilla.mozilla.org/show_bug.cgi?id=1283385) -  Implement UI for \<input type="date"\>
```

Comment:
The `[r?]` tag should be moved to the next line so there won't be a diff on this line.